### ISSUE TITLE
Display format

### ISSUE THREAD

**Issue by shantanuthatte** [walmik/timer.jquery#36]
It would be great if we can define the format of the displayed text.

**Comment by walmik** [comment]
Do you mean have the option to show a digital timer as well? For eg `00:15:30` instead of `15min 30sec` ?

**Comment by shantanuthatte** [comment]
Example:

```javascript
$("#myTimer").timer({
	seconds: 100000,
	format: "%d days %h hr(s) %m min(s) %s sec(s)"
});
```

The ability to define the format the user wants to show. So for a digital one it could be:
```javascript
$("#myTimer").timer({
	seconds: 10000,
	format: "%h:%m:%s"
});
```
Or for leading zeros:
```javascript
$("#myTimer").timer({
	seconds: 100000,
	format: "%H:%M:%S"
});
```

**Comment by walmik** [comment]
This is an excellent idea. Marking it as an enhancement.

**Comment by shantanuthatte** [comment]
MR at #37

**Comment by shantanuthatte** [comment]
Implemented with 6d4959fad4a5ea845d8915502f0db32d04fedba3
